fe/lib/components/LocationSuggest/LocationSuggest.tsx

```diff
@@ -95,6 +98,9 @@ export const LocationSuggest = forwardRef<HTMLInputElement, Props>(
     const [detectLocationError, setDetectLocationError] = useState<string>();
 
     const [placeholder, setPlaceholder] = useState('');
+    const [initialLocation, setInitialLocation] = useState<
+      Location | undefined
```

Comment:
FWIW you can actually omit `| undefined` as `useState` can infer it from the initial state